boards/adi/max32690evkit/max32690evkit_max32690_m4_defconfig

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2023-2024 Analog Devices, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable HEX output, requires for west flash
+CONFIG_BUILD_OUTPUT_HEX=y
+
+# Enable GPIO
+CONFIG_GPIO=y
+
+CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=120000000
```

Comment:
why not pull this from DT? See e.g. GD32

Reply:
Because i see most vendors applied this method.
Thanks for the reference, apply it. see: [Kconfig.defconfig.max32690](https://github.com/zephyrproject-rtos/zephyr/blob/2891de7d6bae8ff7dd0b85b61563bd019bb7e6d3/soc/adi/max32/Kconfig.defconfig.max32690#L8)